doc/release-notes-11413.md

```diff
@@ -0,0 +1,5 @@
+Low-level RPC changes
```

Comment:
nit, I may be wrong but I think you could increment `doc/release-notes-14454.md`?

Reply:
Hm. Does it matter? I assume the files are merged together at the end anyway.